src/components/header/NavButton/NavButton.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+export const NavButton = (
```

Comment:
I think it was because of default classes https://github.com/trussworks/react-uswds/pull/102#discussion_r422313769

Reply:
Ohh, sorry, I missed that somehow.

Reply:
I didn't use the existing button component because it always add the `usa-button` class. That is why I made a generic navButton. I don't mind making two reusable navbuttons components if you think that is better.

Reply:
Yeah sorry I missed that about the `usa-button` class. I think making the reusable nav button components would be good since they are very specific